ls1intum/Artemis
# `Programming exercises`: Fix issues with plagiarism checks

Fixes #4448

### Checklist
#### General


#### Server

#### Changes affecting Programming Exercises

### Motivation and Context


I've updated JPlag to it's current version
* https://github.com/jplag/JPlag/actions/runs/1548612101
* https://github.com/jplag/JPlag/commit/e46aa72cbcc98c4ef70f85e22cf9f619bdab6076

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2..n Students

1. Create some programming submissions (create some similar submissions (they have to be different to the base code / template))
2. As instructor check for plagiarism
3. See whether everything works as expected

### Review Progress





#### Code Review
#### Manual Tests
